src/GraphUtils.ts

```diff
@@ -137,24 +135,24 @@ export function decodeData(data: GraphSketcherState, canvasProperties: CanvasPro
     }
 
     return { curves: normalisedCurves, canvasWidth: canvasProperties.widthPx, canvasHeight: canvasProperties.heightPx };
-};
+}
 
 // TODO 'e' is probably a mouse event of some sort
 export function detect(e: MouseEvent, x: number, y: number) {
-    let mousePosition = getMousePt(e);
+    const mousePosition = getMousePt(e);
     return (getDist(mousePosition, createPoint(x, y)) < 5);
-};
+}
 
-export function getMousePt(e: any) {
-    let x = e.clientX + (window.TouchEvent && e instanceof TouchEvent ? 5 : 0);
-    let y = e.clientY + (window.TouchEvent && e instanceof TouchEvent ? 5 : 0);
+export function getMousePt(e: MouseEvent | Touch) {
+    const x = e.clientX + (window.Touch && e instanceof Touch ? 5 : 0);
+    const y = e.clientY + (window.Touch && e instanceof Touch ? 5 : 0);
```

Comment:
A TouchEvent is a list of current taps on the screen, so has no .clientX/Y. A Touch represents a single instance of one tap, so is the correct class to be using here